libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/modifiers/Clickable.kt

```diff
@@ -17,3 +21,11 @@ fun Modifier.clickableIfNotNull(onClick: (() -> Unit)? = null): Modifier = then(
         Modifier
     }
 )
+
+fun Modifier.niceClickable(
+    onClick: () -> Unit,
+): Modifier {
+    return clip(RoundedCornerShape(4.dp))
+        .clickable { onClick() }
+        .padding(horizontal = 4.dp)
+}
```

Comment:
Maybe `roundedClickable` instead?

Reply:
Yes, your name is better. Will rename on a later PR, do not want to trigger the CI for this. Thanks!